core-bundle/src/Resources/contao/library/Contao/Controller.php

```diff
@@ -1433,6 +1433,11 @@ public static function sendFileToBrowser($strFile, $inline=false)
 	 */
 	public static function loadDataContainer($strTable, $blnNoCache=false)
 	{
+		if ($blnNoCache)
```

Comment:
I would prefer `if (func_num_args() > 1)` here, because using the second argument at all is deprecated and not just setting it to `true`, right? See https://3v4l.org/k07Yn

Reply:
done